lognui.css

```diff
@@ -0,0 +1,599 @@
+}
+
+.alert+.info {
+    background-color: #2196F3;
+    background-image: linear-gradient(to top left, #187dcf, #104977);
+}
+
+.alert+.warning {
+    background-color: #ff9800;
+    background-image: linear-gradient(to top left, #ff9800, #8a570b);
+}
+
+/* Alerts section : END*/
+/* Badges section : START*/
+
+.avatar-container {
+    display: inline-block;
+    position: relative;
+}
+
+.badge {
+    position: absolute;
+}
+
+.badge-round {
+    border-radius: 50%;
+    height: 0.8rem;
+    width: 0.8rem;
+}
+
+.badge-border {
+    border: 2px solid white;
+}
+
+.badge-bottom {
+    bottom: 0em;
+    right: 0em;
+}
+
+.badge-top {
+    top: 0em;
+    right: 0.2em;
+}
+
+.badge-active {
+    background-color: #10b981;
+}
+
+.badge-inactive {
+    background-color: wheat;
+}
+
+/* Badges section : END*/
+/* Buttons section : START*/
+
+.button {
+    padding: 1rem 1.5rem;
+    border-radius: 0.8rem;
+    border: none;
+    cursor: pointer;
+    outline: none;
+}
+
+.button-round {
+    border-radius: 50%;
+}
+
+.primary-button {
+    background-color: var(--primary-color);
+    color: #374151;
+    border: 1px;
+}
+
+.secondary-button {
+    background-color: transparent;
+    color: var(--outline-color);
+    border-color: var(--outline-color);
+    border: 1px solid;
+}
+
+.floating-action-button {
+    position: fixed;
+    right: 4rem;
+    bottom: 2rem;
+    z-index: 1;
+    background-color: var(--primary-color);
+    color: var(--primary-color);
+    padding: 1rem;
+    border: 1px solid var(--outline-color);
+    cursor: pointer;
+    font-size: 1rem;
+    outline: none;
+}
+
+/* Buttons section : END*/
+/* Cards section : START*/
+.card-div {
+    position: relative;
+    display: grid;
+    grid-template-areas:
+        "h1 h1 h1"
+        "card-badge-div card-dismiss-div card-horizontal-div"
+        "card-shadow-div card-text-overlay-div card-text-div";
+    gap: 1rem;
+}
+
+.card {
+    border: 1px solid;
+    position: relative;
+    left: 1rem;
+    top: 1rem;
+    padding: 1rem;
+    margin: 1rem;
+    background-color: white;
+    min-width: 230px;
+}
+
+.card-div>h1 {
+    grid-area: h1;
+}
+
+.card>img {
+    border: 1px solid var(--outline-color);
+    border-radius: 1rem;
+
+}
+
+.card-image {
+    width: 100%;
+}
+
+.card-badge {
+    position: absolute;
+    top: 2em;
+    left: 0em;
+    background-color: var(--primary-color);
+    color: #151d2e;
+    border-radius: 0.2rem;
+}
+
+.card-remove {
+    position: absolute;
+    top: 1rem;
+    right: 1rem;
+    padding: 0.5rem 0.5rem;
+    border: none;
+    cursor: pointer;
+    outline: none;
+    font-weight: bold;
+    font-size: 1rem;
+    background-color: transparent;
+}
+
+.card-overlay {
+    grid-area: card-overlay;
+    position: absolute;
+    max-height: 5rem;
+}
+
+.card-badge-div {
+    grid-area: card-badge-div;
+    position: relative;
+}
+
+.card-dismiss-div {
+    grid-area: card-dismiss-div;
+    position: relative;
```

Comment:
You don't need to specify ```position:relative``` here, It makes no difference if I remove it. Please correct it everywhere.